src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -579,4 +555,19 @@ private boolean isAnyNodeHandshaking(Jedis node) {
     return false;
   }
 
+  private HostAndPort generateHostAndPort(List<Object> hostInfos) {
+    return new HostAndPort(SafeEncoder.encode((byte[]) hostInfos.get(0)),
+            ((Long) hostInfos.get(1)).intValue());
+  }
+
+  private String getNodeId(Jedis node) {
```

Comment:
@marcosnils I will remove it. first, I tried to get node id from info command but it failed.